src/_i18n/en.yml

```diff
@@ -280,6 +280,11 @@ clients:
    expert_advice_title: Expert Advice
    training_and_coaching_title: Training & Coaching
    header: We are helping our clients create quality solutions.
+   adevinta_description: Adevinta have contracted Codurance to help deliver technical capabilities for their digital transformation following the separation of Schibsted and the establishment of Adevinta as a separate classified media business. The digital strategy involves all digital assets to be running independently of each other by the end of 2019, and our embedded craftspeople form part of the team to help deliver these critical business goals.
```

Comment:
Should these descriptions be written in the past tense? 
If so, the Adevinta description would need a little adjustment.